### DISCUSSION TITLE
Orleans for real-time model scoring, consuming from Kafka (better alternative to Spark streaming)

### DISCUSSION THREAD

**Discussion by fwaris** [dotnet/orleans#8277] [Show and tell]
Sharing some recent experience:

[Applying Some 'Unconventional Wisdom'​ to Improve Model Scoring wrt. High-Velocity Streaming Data](https://www.linkedin.com/pulse/applying-some-unconventional-wisdom-improve-model-wrt-waris-phd)

I think the potential for using Orleans for real-time model scoring should be more appreciated.

**Comment by bradygaster** [comment]
This is an excellent article and was rewarding to read. Did you share this in the discord, too?

**Reply by fwaris** [reply]
@bradygaster thanks. I would be happy to share but I don't use discord. Feel free to post it on discord as you see fit.
